docs/_data/navbar.yml

```diff
@@ -0,0 +1,30 @@
+# Navbar menu navigation links
+  - title: Home
+    url: /
+  - title: Docs
+    url: /docs/getting-started
+#     <-- example on how to create navigation dropdown -->
```

Comment:
We can leave the example here, but is it intentional?

Reply:
I moved the example to the docs/readme.md file